src/pysony.py

```diff
@@ -38,12 +38,14 @@
 # Improved with code from 'https://github.com/storborg/sonypy' under MIT license.
 
 class ControlPoint(object):
-    def __init__(self, addr=SSDP_ADDR, port=SSDP_PORT):
+    def __init__(self, addr=SSDP_ADDR, port=SSDP_PORT, interface=None):
         self.addr, self.port = addr, port
         sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
         sock.settimeout(0.1)
         # Set the socket to broadcast mode.
         sock.setsockopt(socket.IPPROTO_IP, socket.IP_MULTICAST_TTL, 2)
+        if interface:
+          sock.setsockopt(socket.SOL_SOCKET, 25, str(interface + '\0').encode('utf-8'))
```

Comment:
Extremely minor nitpick: Everything else in this file looks like it uses 4 spaces for indentation, but you've only got 2 spaces here.

What does `25` mean in the `setsockopt` call?

Reply:
please use 4 spaces like other line.

is that 25 is optname...? 
socket.setsockopt(level, optname, value: int)
socket.setsockopt(level, optname, value: buffer)
socket.setsockopt(level, optname, None, optlen: int)

Reply:
I happened to dig into this myself the other day, 25 is indeed `optname` and at least in terms of Linux based sockets, it refers to `SO_BINDTODEVICE` as seen here: https://github.com/torvalds/linux/blob/169e77764adc041b1dacba84ea90516a895d43b2/include/uapi/asm-generic/socket.h